pkg/imgpkg/bundle/locations_config.go

```diff
@@ -0,0 +1,72 @@
+// Copyright 2020 VMware, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+package bundle
+
+import (
+	"fmt"
+	"io/ioutil"
+
+	"sigs.k8s.io/yaml"
+)
+
+const (
+	LocationFilepath   = "images-locations.yml"
+	LocationKind       = "Locations"
+	LocationAPIVersion = "imgpkg.carvel.dev/v1alpha1"
+)
+
+type LocationConfig struct {
+	APIVersion string          `json:"apiVersion"` // This generated yaml, but due to lib we need to use `json`
+	Kind       string          `json:"kind"`       // This generated yaml, but due to lib we need to use `json`
+	Images     []ImageLocation `json:"images"`     // This generated yaml, but due to lib we need to use `json`
+}
+
+type ImageLocation struct {
+	Image string `json:"image"` // This generated yaml, but due to lib we need to use `json`
+	// Repository will not be used right now because all images will be in the bundle repository
+	Repository string `json:"repository,omitempty"` // This generated yaml, but due to lib we need to use `json`
+	Bundle     bool   `json:"bundle"`               // This generated yaml, but due to lib we need to use `json`
```

Comment:
wdyt about calling this `IsBundle  bool  json:"isBundle"`. imho is makes it more understandable that this is a t/f kinda a thing vs some bundle configuration.

Reply:
done